dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/ProcessDefinitionService.java

```diff
@@ -67,6 +70,15 @@ Map<String, Object> createProcessDefinition(User loginUser,
                                                 String otherParamsJson,
                                                 ProcessExecutionTypeEnum executionType);
 
+    /**
+     * create process definition V2
+     *
+     * @param loginUser login user
+     * @param workflowCreateRequest the new workflow object will be created
+     * @return New ProcessDefinition object created just now
+     */
+    ProcessDefinition createProcessDefinitionV2(User loginUser, WorkflowCreateRequest workflowCreateRequest);
```

Comment:
What's the different between v1 and v2? Maybe v2 it's not a good guide for developers if v1 can not be replaced with v2.

Reply:
v1 is for currently API(operate workflow and task and task relation object), and V2 is only operate workflow object. V2 is mainly for RESTful api, which we only change one single object instand of too many object

Reply:
Will we use v2 uniformly in the future? including UI or support restful api or pyds?

Reply:
I hope we can do that, and in that moment, maybe it it the best time to migrate python api into separate repository